client/msp/tenants/add.go

```diff
@@ -0,0 +1,25 @@
+package tenants
+
+import (
+	"context"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/http"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/url"
+)
+
+func AddExistingTenantUsingApiToken(ctx context.Context, client http.Client, addInp MspAddExistingTenantInput) (*MspTenantOutput, *CreateError) {
+	client.Logger.Println("Creating tenant for CDO")
```

Comment:
I suggest updating the comment, as we're only adding the tenant not creating it

Reply:
Ouch, good catch, mate. Fixing.